<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Propertiesdialog</class>
 <widget class="QWidget" name="Propertiesdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="xspinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="yspinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textColorLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="outlineColorLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundColorLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="textColorButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>字体色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outlineColorButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>底线色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backgroundColorButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>背景色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>坐标设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
